test/system-test/system-tests/pkcs11_system_test.c

```diff
@@ -33,7 +33,7 @@
 #include "core_pkcs11.h"
 #include "core_test_pkcs11_config.h"
 
-/* Logging includes. */
+/* Loggign includes. */
```

Comment:
spelling... Say, how did the PR spelling check pass?

Reply:
I need to update the spell check script to include this file. I will do that in a separate PR.